lib/modifiers.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Metrix.Modifiers do
```

Comment:
I like the idea of having a single Agent handle all global state (context and prefix). If you want to do that, I'd be 👍 

Wonder if it should be a separate PR so we don't conflate things?